contents/uix/uix.md

```diff
@@ -0,0 +1,47 @@
+# UI/UX
+
+Author: [Ang Shi Ya](https://github.com/AngShiYa)
+
+## Overview
+
+User Interface (UI) and User Experience (UX) are terms often used interchangeably. In fact, These two terms are quickly becoming dangerous buzzwords that are used inaccurately, creating massive confusion in the industry. A simple google search will show that UI Designer and UX Designer jobs are listed under similar titles and descriptions. 
+
+Learning the key differences between UI and UX can not only help us identify and settle down for the correct job role, but can also help us apply each of these concepts appropriately to our own applications. This document will address the difference between UI and UX as well as some of the techniques used in UI Design and UX Design.
```

Comment:
I suggest rewriting this sentence in this way: "In addition to helping us to identify and *apply* for the right role, understanding the key differences between UI and UX helps us in using the right concept in our own applications."